CMakeLists.txt

```diff
@@ -2,6 +2,19 @@
 project(Halide)
 cmake_minimum_required(VERSION 3.2)
 
+set(CPACK_PACKAGE_VENDOR "Halide")
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/LICENSE.txt")
+set(CPACK_MONOLITHIC_INSTALL OFF)
+if (WIN32)
+  set(CPACK_GENERATOR "ZIP")
+else()
+  set(CPACK_GENERATOR "TGZ")
+endif()
+# Remove this to get package names that are formatted as
+# ${CPACK_PACKAGE_NAME}-${CPACK_PACKAGE_VERSION}-${CPACK_SYSTEM_NAME}.
```

Comment:
I don't quite understand the difference between these options; why would one want the commented-out variant?

Reply:
The commented out variant starts being useful when there's automation in place to create packages from release branches/tags.

`Halide-v1.0.0-linux.tgz` and `Halide-v1.0.0-macos.tgz` vs. just one `Halide.tgz`.